src/python/pants/option/global_options.py

```diff
@@ -44,7 +44,8 @@ def register_bootstrap_options(cls, register):
     register('-l', '--level', choices=['debug', 'info', 'warn'], default='info', recursive=True,
              help='Set the logging level.')
     register('-q', '--quiet', type=bool, recursive=True,
-             help='Squelches most console output.')
+             help='Squelches most console output. NOTE: if this value is specified via a command line option '
```

Comment:
Maybe:
```
"NOTE: Some tasks default to behaving quietly: inverting this option supports making them noisier than they would be otherwise."
```

The environment variable / command line / config file is implicit, I think.

Reply:
done.